docs/csharp/whats-new/tutorials/generate-consume-asynchronous-stream.md

```diff
@@ -138,7 +138,7 @@ You use another extension method, <xref:System.Threading.Tasks.TaskAsyncEnumerab
 
 :::code language="csharp" source="snippets/generate-consume-asynchronous-streams/finished/Program.cs" id="SnippetEnumerateWithCancellation" :::
 
-You can get the code for the finished tutorial from the [dotnet/docs](https://github.com/dotnet/docs) repository in the [csharp/tutorials/AsyncStreams](https://github.com/dotnet/docs/tree/main/docs/csharp/tutorials/snippets/generate-consume-asynchronous-streams/finished) folder.
+You can get the code for the finished tutorial from the [dotnet/docs](https://github.com/dotnet/docs) repository in the [csharp/whats-new/tutorials](https://github.com/dotnet/docs/tree/main/docs/csharp/whats-new/tutorials/snippets/generate-consume-asynchronous-streams/finished) folder.
```

Comment:
Is there a tool you run to find these? I'll run it as I continue re-organizing content. (Otherwise, I'll make a note to check for download links when I move samples.

Reply:
These are found by the OPS "Broken links" tool.

![image](https://user-images.githubusercontent.com/24882762/113594237-43f6cb80-95ec-11eb-926c-66baa376b74a.png)